i3lock.c

```diff
@@ -58,6 +58,8 @@ static bool beep = false;
 bool debug_mode = false;
 static bool dpms = false;
 bool unlock_indicator = true;
+static bool capslock_indicator = false;
```

Comment:
Why is this disabled by default? I think it makes more sense to enable it by default.